purchase_order_qty_change_no_recompute/models/purchase_order.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2021 Tecnativa - Víctor Martínez
+# Copyright 2021 Tecnativa - Pedro M. Baeza
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import models
+
+
+class PurchaseOrderLine(models.Model):
+    _inherit = "purchase.order.line"
+
+    def _onchange_eval(self, field_name, onchange, result):
+        """Remove the trigger for the undesired onchange method with this field.
+        We have to act at this place, as `_onchange_methods` is defined as a
+        property, and thus it can't be inherited due to the conflict of
+        inheritance between Python and Odoo ORM, so we can consider this as a HACK.
+        """
+        if field_name == "product_qty":
```

Comment:
Please add a safeguard for tests evaluating `config["test_enable"]` and only acting if a special context key is passed.

Reply:
Do you think we should apply this change also to the `sale_order_qty_change_no_recompute` addon?

Reply:
Yeah, that would prevent the problem you have with the other module in tests.